### ISSUE TITLE
Issues with direcotry template in 2014-09-29 "Hrun"?

### ISSUE THREAD

**Issue by thorst** [jvanopstal/template-by-name#5]
Im running the newest version of Doku- 2014-09-29 "Hrun", and having some issues, with what I think should be pretty easy.

I can get the file template to work `__yourfilename.txt` but I cant get the directory template to work `~~yourdirectoryname.txt`. No matter where I put it, it doesnt seem to catch it.
- `MyProjectName`
  - `~~Solutions.txt` => _Not Working_ => template for `Project 1.txt` under `Solution 1`
  - `Solutions`
    - `__start.txt` => _Working_ => template for the `start.txt` under `Solution 1`
    - `Solution 1`
      - `start.txt`
      - `Project 1.txt`

Basically I want all files under the `Solutions` directory to inherit from a template UNLESS it is named start. If its named start then I want it to inherit from another template.

Am I doing something wrong, or is there a compatibility issue?

**Comment by blackbinary-net** [comment]
i have the same problem :(
